pages/_app.tsx

```diff
@@ -1,8 +1,38 @@
-import { AppProps } from 'next/app'
-import '../styles/index.css'
+import { AppProps } from "next/app";
+import "../styles/index.css";
+import Router from "next/router";
+import { PageLoader } from "../components/PageLoader";
+
+import { AnimatePresence } from "framer-motion";
+import { useEffect, useState } from "react";
 
 function MyApp({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />
+  const [loading, setLoading] = useState(false);
+  useEffect(() => {
+    // Used for page transition
+    const start = () => {
```

Comment:
can rename it to startLoader and instead of end stopLoader

Reply:
Hey @sahadat-sk, I really appreciate the effort you've put into the hover button – it's a cool feature! I think the preview card itself is quite intuitive and users will naturally know to click on it for more info. Maybe we can do without the extra button to keep things sleek. What do you think?